packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -1001,6 +1001,14 @@ export const Table = <
                       column.minWidth = columnHasExpanders
                         ? COLUMN_MIN_WIDTHS.withExpander
                         : COLUMN_MIN_WIDTHS.default;
+
+                      if (
+                        column.width &&
```

Comment:
very small suggestion: this `column.width` check can be removed, since it will be automatically covered by `typeof column.width === 'number'`

Reply:
Removed in this [commit](https://github.com/iTwin/iTwinUI/pull/2138/commits/f7f8548bb8c105617130c109185382b8f8761126).